From a5ee11e0258967c3e2ce04841b423b41cc4cd24f Mon Sep 17 00:00:00 2001
From: Kiran Kumar Lokere <quic_klokere@quicinc.com>
Date: Sun, 19 May 2024 22:07:17 -0700
Subject: [PATCH] Add new traffic type values for flow report vendor attribute

Define new traffic type values for
QCA_WLAN_VENDOR_ATTR_FLOW_STATS_TRAFFIC_TYPE attribute.

Signed-off-by: Jouni Malinen <quic_jouni@quicinc.com>
---
 src/common/qca-vendor.h | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/src/common/qca-vendor.h b/src/common/qca-vendor.h
index 5c4596fde..3161cbe61 100644
--- a/src/common/qca-vendor.h
+++ b/src/common/qca-vendor.h
@@ -17338,12 +17338,18 @@ enum qca_wlan_vendor_attr_ap_suspend {
  * @QCA_TRAFFIC_TYPE_GAMING: Traffic type is gaming
  * @QCA_TRAFFIC_TYPE_VOICE_CALL: Traffic type is a voice call
  * @QCA_TRAFFIC_TYPE_VIDEO_CALL: Traffic type is a video call
+ * @QCA_TRAFFIC_TYPE_SCREEN_SHARE: Traffic type is screen share
+ * @QCA_TRAFFIC_TYPE_UNKNOWN: Traffic type is unknown
+ * @QCA_TRAFFIC_TYPE_INVALID: Invalid traffic type
  */
 enum qca_traffic_type {
 	QCA_TRAFFIC_TYPE_STREAMING = 0,
 	QCA_TRAFFIC_TYPE_GAMING = 1,
 	QCA_TRAFFIC_TYPE_VOICE_CALL = 2,
 	QCA_TRAFFIC_TYPE_VIDEO_CALL = 3,
+	QCA_TRAFFIC_TYPE_SCREEN_SHARE = 4,
+	QCA_TRAFFIC_TYPE_UNKNOWN = 5,
+	QCA_TRAFFIC_TYPE_INVALID = 6,
 };
 
 /**
-- 
2.43.0

